packages/common/src/compiler/exploreCompiler.test.ts

```diff
@@ -103,3 +104,47 @@ describe('Default field labels render for', () => {
         expect(friendlyName('my_1field_id')).toEqual('My 1field id');
     });
 });
+
+describe('Compile metrics with filters', () => {
+    beforeAll(() => {
+        jest.useFakeTimers();
+        jest.setSystemTime(new Date('04 Apr 2020 00:12:00 GMT').getTime());
```

Comment:
nice, could we use it to simulate different timezones ?

Reply:
I'm not sure. I don't think this changes the server timezone. 
btw, I copied this from an existing test.